diff --git a/src/vidstab.mk b/src/vidstab.mk
index 22f35aa7..59cecaee 100644
--- a/src/vidstab.mk
+++ b/src/vidstab.mk
@@ -10,7 +10,7 @@ $(PKG)_GH_CONF  := georgmartius/vid.stab/tags,v
 $(PKG)_DEPS     := cc
 
 define $(PKG)_BUILD
-    cd '$(BUILD_DIR)' && $(TARGET)-cmake '$(SOURCE_DIR)'
+    cd '$(BUILD_DIR)' && $(TARGET)-cmake -DUSE_OMP=OFF '$(SOURCE_DIR)'
     $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
     $(MAKE) -C '$(BUILD_DIR)' -j 1 install
 
